Makefile

```diff
@@ -1,35 +1,40 @@
-CFLAGS+=$(shell pkg-config --cflags cairo-xlib xinerama glib-2.0 xext x11 xtst 2> /dev/null || echo -I/usr/X11R6/include -I/usr/local/include)
-LDFLAGS+=$(shell pkg-config --libs cairo-xlib xinerama glib-2.0 xext x11 xtst 2> /dev/null || echo -L/usr/X11R6/lib -L/usr/local/lib -lX11 -lXtst -lXinerama -lXext -lglib)
-LDFLAGS+=$(shell pkg-config --libs glib-2.0)
+CFLAGS+=$(shell pkg-config --cflags cairo-xlib 2> /dev/null)
```

Comment:
Why split these up?

Reply:
Not sure, that wasn't my change.  It's been a *long* time since I've done much makefiles and C, so not sure what makes sense here.  It works fine for me either way.